pkg/backup/restore_online.go

```diff
@@ -818,6 +821,11 @@ func (r *restoreResumer) doDownloadFilesWithRetry(
 			return nil
 		}
 		log.Warningf(ctx, "failed attempt #%d to download files: %v", rt.CurrentAttempt(), err)
```

Comment:
this attempt number is gonna be bogus once we're resetting so I might remove it from the log message since it could be confusing otherwise.